jck/jtrunner/JavatestUtil.java

```diff
@@ -489,6 +489,9 @@ private static boolean generateJTB() throws Exception {
 			} else if (spec.contains("alpine-linux")) {
```

Comment:
can we do it in one line?
```
else if (spec.contains("alpine-linux") || spec.contains("riscv64")) {
```

Reply:
Yes we can if you think that's best - that was one of the two alternative options that I put in the initial description as I wasn't clear if it has been deliberately set up with only one condition in each branch of the if/else section :-)

Reply:
Ultimately in a future revision, this code can be improved/updated to check if a platform under test is headless, and set 'no' to robot.